rust/src/cliwrap.rs

```diff
@@ -61,10 +61,13 @@ pub fn entrypoint(args: &[&str]) -> Result<()> {
     let args = &args[1..];
 
     let host_type = crate::get_system_host_type()?;
+    let skip_wrap = std::env::var("RPMOSTREE_CLIWRAP_SKIP").unwrap_or("none".to_string());
```

Comment:
I think this would be a bit cleaner as a `bool`, like:
```suggestion
    let skip_wrap = std::env::var("RPMOSTREE_CLIWRAP_SKIP").is_some();
```

We would accept literally any value there as being `true`; I don't see a reason to support e.g. `RPMOSTREE_CLIWRAP_SKIP=false`.